### PR TITLE
US004: implement CRUD for ParkingZone model

### PR DESCRIPTION
**Pull request by dilmurod-rozikov**

### CODE REVIEW

File: ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        }
+
+        // POST: Admin/ParkingZones/Create
+        // To protect from overposting attacks, enable the specific properties you want to bind to.
+        // For more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create([Bind("Id,FirstName,LastName,Address,CreatedDate")] ParkingZone parkingZone)
+        {
+
```
Comment (abdullokhtolibjonov) [inline ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs:54]: remove this line

File: ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                    _context.Update(parkingZone);
+                    await _context.SaveChangesAsync();
+                }
+                catch (DbUpdateConcurrencyException)
+                {
+                    if (!ParkingZoneExists(parkingZone.Id))
+                        return NotFound();
+                    else
+                        throw;
+
```
Comment (abdullokhtolibjonov) [inline ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs:102]:
```suggestion
                        throw;
```

Comment (abdullokhtolibjonov) [reply ParkingZoneApp/Areas/Admin/Controllers/ParkingZonesController.cs:102]: unnecessary line

File: ParkingZoneApp/Areas/Admin/Models/ParkingZone.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        [Key]
+        [Required]
+        public Guid Id { get; set; }
+
+        [Required]
+        public string FirstName { get; set; }
+
+        [Required]
+        public string LastName { get; set; }
```
Comment (abdullokhtolibjonov) [inline ParkingZoneApp/Areas/Admin/Models/ParkingZone.cs]: Parking zone has FirstName and LastName? remove them just add Name

File: ParkingZoneApp/Areas/Admin/Views/ParkingZones/Delete.cshtml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@model ParkingZoneApp.Areas.Admin.Models.ParkingZone
+
+@{
+    ViewData["Title"] = "Delete";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+
```
Comment (abdullokhtolibjonov) [inline ParkingZoneApp/Areas/Admin/Views/ParkingZones/Delete.cshtml:8]: remove this line

File: ParkingZoneApp/Areas/Admin/Views/ParkingZones/Details.cshtml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+﻿@model ParkingZoneApp.Areas.Admin.Models.ParkingZone
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+@{
+    ViewData["Title"] = "Details";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+
```
Comment (abdullokhtolibjonov) [inline ParkingZoneApp/Areas/Admin/Views/ParkingZones/Details.cshtml:7]: unnecessary line

**Review by abdullokhtolibjonov — CHANGES_REQUESTED** [review]

File: ParkingZoneApp/Areas/Admin/Views/ParkingZones/Delete.cshtml (hunk 4 above)
Comment (dilmurood) [reply ParkingZoneApp/Areas/Admin/Views/ParkingZones/Delete.cshtml:8]: ok  but this line remaining @{Layout = "~/Views/Shared/_Layout.cshtml";}

File: ParkingZoneApp/Areas/Admin/Models/ParkingZone.cs (hunk 3 above)
Comment (dilmurood) [reply ParkingZoneApp/Areas/Admin/Models/ParkingZone.cs]: sorry for misunderstanding

**Pull request closed without merging**